cli/src/lib.rs

```diff
@@ -583,6 +588,34 @@ fn new_program(name: &str) -> Result<()> {
     Ok(())
 }
 
+fn rename_program(from: &str, to: &str) -> Result<()> {
+    rename_in_file(&Path::new(&format!("programs/{}/Cargo.toml", from)), &from, &to)?;
+    rename_in_file(&Path::new(&format!("programs/{}/src/lib.rs", from)), &from, &to)?;
+    fs::rename(&format!("programs/{}", from), &format!("programs/{}", to))?;
+    
+    // Users probably have either .js or .ts tests, not both. Don't panic.
+    rename_in_file(&Path::new(&format!("tests/{}.js", from)), &from, &to).ok();
```

Comment:
why convert to an option if youre not gonna do anything with it?

Reply:
seemed like a decent way to tell compiler not to warn, but maybe `let _ = func();` is better here.

Reply:
yes that seems more idiomatic